src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1236,9 +1267,23 @@ static fsal_status_t kvsfs_getattrs(struct fsal_obj_handle *obj_hdl,
 
 out:
 	T_EXIT0(result.major);
+    perfc_trace_attr(PEA_GETATTR_RES_MAJ, result.major);
```

Comment:
Would it be prudent to add tracepoints for any other attributes like members of fsal_obj_handle?

Reply:
As of now, ADDB traces for integer variable can be logged.